CHANGELOG.md

```diff
@@ -7,9 +7,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ### Changed
 * **BREAKING**: Tertiaire button `.btn-link` redesign (#372).
+* **BREAKING**: SVG iconen (#414).
 
 #### Migratie notities:
 
+##### SVG Iconen (#414)
```

Comment:
Moet er in het changelog (of elders in de documentatie) wellicht nog iets (meer) over de package svg4everybody ?

Reply:
Zoals ik al aangaf:

>https://github.com/dso-toolkit/dso-toolkit/pull/413 ik heb je als reviewer toegevoegd aan het PR. ik moet nog changelog en releasenotes schrijven maar inhoudelijk is dit 't wel.. wil jij het doornemen?